docarray/typing/tensor/torch_tensor.py

```diff
@@ -137,7 +137,7 @@ def _docarray_to_json_compatible(self) -> np.ndarray:
         Convert `TorchTensor` into a json compatible object
         :return: a representation of the tensor compatible with orjson
         """
-        return self.numpy()  ## might need to check device later
+        return self.detach().numpy()  # might need to check device later
```

Comment:
This only seems to tackle json serialization. Do al the other ones work? Protobuf, pickle, etc?